### PR TITLE
Added latmem package to buildroot

### PR DESCRIPTION
**Pull request by tudang-wdc**

### CODE REVIEW

File: software/patches/buildroot/latmem.patch [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+ create mode 100644 package/latmem/Config.in
+ create mode 100644 package/latmem/latmem.mk
+ create mode 100644 package/latmem/src/Makefile
+ create mode 100644 package/latmem/src/latmem.c
+
+diff --git a/package/Config.in b/package/Config.in
+index f72c77b416..e4a79ece5a 100644
+--- a/package/Config.in
++++ b/package/Config.in
+@@ -1871,6 +1871,7 @@ menu "Miscellaneous"
```
Comment (atishp04) [inline software/patches/buildroot/latmem.patch]: It should be put under benchmark not Miscellaneous.

File: software/patches/buildroot/latmem.patch [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
++
++$(eval $(generic-package))
+diff --git a/package/latmem/src/Makefile b/package/latmem/src/Makefile
+new file mode 100644
+index 0000000000..94ca525859
+--- /dev/null
++++ b/package/latmem/src/Makefile
+@@ -0,0 +1,17 @@
++export PATH:=$(HOME)/Workspace/riscv64--glibc--bleeding-edge-2018.11-1/bin:$(PATH)
++CC=riscv64-linux-
```
Comment (atishp04) [inline software/patches/buildroot/latmem.patch]: I don't think you need to explicitly export PATH or CC for individual package. Buildroot should take care of that.
It also may not work on other machines as toolchain prefix or toolchain install path may be different.

**Review by atishp04 — CHANGES_REQUESTED** [review]
Otherwise, looks good.

**Comment by tudang-wdc** [comment]
Hi Atish,

I added some changes to your request. Please merge if the changes look good.

**Review by atishp04 — APPROVED** [review]
FYI: You can squash any fixes that address the comments and just update the branch.

I am merging this PR. Please update the rootfs file with latmem benchmark as a separate PR.

**Pull request merged**